CardData.py

```diff
@@ -112,15 +103,9 @@ def create_card_json(gwent_data_helper, patch):
         card['categories'] = []
         card['categoryIds'] = []
         categoriesSum = int(template.find('Categories').find('e0').attrib['V']);
-        # XML Card category is the sum of all categories of the card
-        for category in range(LAST_CATEGORY, 1, -1):
-            categoryPower = 2**category
-            if categoriesSum - categoryPower >= 0:
-                categoriesSum -= categoryPower
-                if category in CATEGORIES:
-                    card['categoryIds'].append("card_category_" + str(category))
-                if categoriesSum == 0:
-                    break
+        for category, bit in enumerate("{0:b}".format(categoriesSum)[::-1]):
+            if bit == '1':
+                card['categoryIds'].append("card_category_{0}".format(category))
```

Comment:
I think index 0 is suppose to be category 1
so `format(index + 1)`

Reply:
I think it's right, it's working for me, I'm getting the correct categories.

The index shouldn't matter since we aren't using the list anymore, the result after the bit mask is solved is the category number.

Reply:
`category` is the index in the array of bits (from 0), but if it works ¯\_(ツ)_/¯

Reply:
The list is reversed though right? So the first category is actually the largest bit?

Reply:
no, the number is read left-to-right (e.g. `2` is `10`)
and `enumerate()` is not reversed, only it's argument